docs/api/security/trusted_storage.md

```diff
@@ -0,0 +1,27 @@
+<h2 id="blockdevice-port">Trusted Storage Reference API</h2>
+
+## PSA Internal Trusted Storage
+
+The PSA Internal Trusted Storage is a PSA Root of Trust Service which provides secure storage of small data items, for example, cryptographic keys.
```

Comment:
```suggestion
The PSA Internal Trusted Storage is a PSA Root of Trust Service which provides key value secure storage in internal flash for small data items. The main use cases for PSA Internal Trusted Storage APIs are:
- provide storage solution for persistent keys in PSA Crypto Service.
- storing rollback protection tokens for external application secure storage.
- storing persistent context in PSA Root of Trust services (i.e. epoch, device unique key. e.t.c.)
```

Reply:
The term "secure" here is ambiguous and unnecessary. 
"which provides key value secure storage in internal flash for small data items"->"which provides key value storage in internal flash for small data items. It provides confidentiality and integrity by preventing non-PSA Root of Trust software from reading from or writing to the storage directly."

Reply:
the later will be true on PSA platforms only - i think it should be mentioned explicitely as we plan to provide these APIs on non-PSA platforms (like k64f) for compatibility reasons

Reply:
good point.

Reply:
Guys, I am applying Alex's suggestion (almost) as is.
Please review one more time to see if acceptable.